sdks/java/extensions/avro/src/test/java/org/apache/beam/sdk/extensions/avro/schemas/utils/AvroUtilsTest.java

```diff
@@ -78,13 +77,37 @@ public class AvroUtilsTest {
   private static final org.apache.avro.Schema NULL_SCHEMA =
       org.apache.avro.Schema.create(Type.NULL);
 
+  private static final String VERSION_AVRO =
+      org.apache.avro.Schema.class.getPackage().getImplementationVersion();
+
+  private Iterable<?> randomData(org.apache.avro.Schema schema, int maxLength) throws Exception {
+    Iterable<?> data;
+    if (VERSION_AVRO.equals("1.8.2")) {
+      data =
+          (Iterable<?>)
+              Class.forName("org.apache.avro.RandomData")
+                  .getDeclaredConstructor(org.apache.avro.Schema.class, Integer.TYPE)
+                  .newInstance(schema, maxLength);
+    } else {
+      data =
+          (Iterable<?>)
+              Class.forName("org.apache.avro.util.RandomData")
+                  .getDeclaredConstructor(org.apache.avro.Schema.class, Integer.TYPE, Boolean.TYPE)
+                  // force Utf8 in random data to match with String type used in AvroUtils
+                  .newInstance(schema, maxLength, true);
```

Comment:
In avro 1.9+, `RandomData` has moved from the `tests` classifier lib into the `util` package of the main library and default behavior, has moved to `String` instead of `Utf8`.